hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -780,9 +767,8 @@ public void postShipEdits(List<Entry> entries, int batchSize) {
     this.manager.getGlobalMetrics().setWALReaderEditsBufferBytes(newBufferUsed);
   }
 
-  @Override
-  public WALFileLengthProvider getWALFileLengthProvider() {
-    return walFileLengthProvider;
+  @Override public WALProvider getWALProvider() {
```

Comment:
I think you'd better update your formatter config to put Override annotation to the line before? I think this is the typical type in our code base, and there is no checkstyle issue for this style.

Reply:
Will do